### PR TITLE
create risk-management.md

### PR DESCRIPTION
**Pull request by Davidutro**
I want to add the question about compensation of risk teams, but don't know how best to answer it clearly since it's in a state of flux currently.

### CODE REVIEW

File: faqs/risk-management.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+*   The Initial Collateral Onboarding Process: this covers the trade support structure, distribution of token holdings and available data series.
+
+*   The Operational Assessment Process: this covers the organization itself, the organization's operational risks, regulatory risks, the quality of the community, etc.
+
+*   The Technological Assessment Process: this covers the robustness and security of the underlying technology.
+
+Due diligence related to each of these three parts will give one information that can be compiled into distinct features and assigned ratings. These features are:
+
+*   Team; Core team and advisors
```
Comment (ghettodev) [inline faqs/risk-management.md]: Please bold definitions, use colons, and complete sentences. 

* **Team:** Core team and advisors.
* **Community:** Sentiment analysis.

File: faqs/risk-management.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+*   Business Models ; Structural and legal analysis
+
+An overall score, formed as a result of the individual ratings, will serve as the mathematical input for the Risk Model. The score, judged against other assets in the portfolio, will be the adjustment factor to the risk parameters of that token.
+
+## What is Exposure Risk?
+
+Exposure Risk refers to the exposure one has to particular collateral and generally whether one has too much.  Exposure Risk is relative to the available supply of the asset and its liquidity and quality rating. For a CDP, if a significant portion of an asset's supply was locked as collateral, its subsequent liquidation could result in significant price impact on the market. The risk to the MakerDAO system is that a price drop of that collateral type may result in the liquidation of increasingly more of the underlying asset in CDPs than the market can handle.
+
+Exposure risk also takes into account the Free Float of the token and its relationship to the token's liquidity. Free Float is the readily traded portion of the available supply. Liquidity in the market is a result of the free float.
```
Comment (ghettodev) [inline faqs/risk-management.md]: inconsistent capitalization: Exposure risk. Please double check the rest of the doc.

File: faqs/risk-management.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+## What is Correlation Risk?
+
+Correlation Risk is having too much exposure to a group of assets whose values are highly correlated to one another. As a portfolio grows to include more uncorrelated assets, the correlation risk decreases.  A generally accepted principle is that a combination of diverse assets will lower the risk of any individual holding. The diversification benefit is maximized with the addition of uncorrelated assets. Correlation risk is defined as the volatility of the diversification benefit.
+
+## What is Price Feed Risk?
+
+Price feed Risk is the risk of a token's pricing information becoming unavailable, inaccurate, or corrupt. This risk is present at two levels, at the source of the price information, and at the [Oracles](https://drive.google.com/a/makerdao.com/open?id=1UxgYFgT5JKmXrdl8zPc7_u8aDMWV2chrcm6ssnkbgvo).
```
Comment (ghettodev) [inline faqs/risk-management.md]: Linking to google docs

**Comment by ghettodev** [comment]
Leave out compensation information for now. Please add updated summary.md file

**Comment by Davidutro** [comment]
@ghettodev Changes made, will update summary once these are committed

**Pull request merged**